src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -862,6 +907,22 @@
     justify-content: center;
     align-items: stretch;
 
+
+
+    input[type='text'] {
```

Comment:
I'm not sure what it is. But the input field doesn't seem properly aligned. It looks too close to the `/` to me and the space between the left angle looks too big. Do you see that as well?

Reply:
Yes! I updated the css and this should be fixed with latest changes.